firmware-common/common2015/utils/rtp.hpp

```diff
@@ -88,14 +88,17 @@ struct RobotStatusMessage {
     static constexpr float BATTERY_READING_SCALE_FACTOR = 0.09884;
     uint8_t battVoltage;
 
-    uint8_t ballSenseStatus : 2;
-
     // 1 bit for each motor - 1 = error, 0 = good
-    uint8_t motorErrors:5;
+    unsigned  motorErrors:5;
+
+    unsigned ballSenseStatus : 1;
+
+    // robot charged?
+    unsigned kickStatus:1;
 
     // 0 = good, 1 = not initialized, 2 = error
-    uint8_t fpgaStatus:2;
-};
+    unsigned fpgaStatus:1;
```

Comment:
I'm not sure about this syntax, but if the `1` is the number of bits, should the comment be changed to 0=good, 1=bad?